code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -1176,6 +1219,7 @@ Pass a positive integer as an argument to override a bot's default speed.
 	data["canhack"] = canhack(user)
 	data["emagged"] = emagged
 	data["remote_disabled"] = remote_disabled
+	data["wrenched"] = wrenched //Are the bot's components wrenched in place ? Used by secbots and griefsky
```

Comment:
Any reason to add this to TGUI?

Reply:
By TGUI do you mean the TGUI the players open with the ID interface ? Or another kind of TGUI ? I added the data here because it was said to be common to all bots and used by another script.

Reply:
all bots use this to send information to UI. Does being wrenched or not change anything for the UI, I see no TGUI / UI changes

Reply:
No it does not change anything to the UI. I assume I remove it since it is not used ?

Reply:
yup
its for sending dm information to ui

Reply:
Okay! That's removed.